xbmc/ServiceManager.cpp

```diff
@@ -23,9 +23,12 @@
 #include "utils/log.h"
 #include "interfaces/AnnouncementManager.h"
 #include "interfaces/generic/ScriptInvocationManager.h"
+#include "addons/binary/interfaces/api1/PVR/AddonCallbacksPVR.h"
 
 bool CServiceManager::Init1()
 {
+  V1::KodiAPI::PVR::CAddonCallbacksPVR::EnableInterface(false);
```

Comment:
I mean that then for every new API level it need to call the same independent for his class. This bring a list of same calls and if for some reason it becomes more and the `EnableInterface`becomes for some reason also called on other place a easy source for errors.